tests/backend_/utils/test_parse_uri.py

```diff
@@ -94,3 +94,11 @@ def test_invalid_credentials(self):
     def test_no_scheme(self):
         with self.assertRaisesMessage(pymongo.errors.InvalidURI, "Invalid URI scheme"):
             parse_uri("cluster0.example.mongodb.net")
+
+
+# TODO: This can be moved to `test_features` once transaction support is merged.
+class ParseUriOptionsTests(TestCase):
+    @skipUnlessDBFeature("supports_queryable_encryption")
```

Comment:
Probably we just make installing `pymongo[encryption]` a required part of running our test suite. In that case, this test wouldn't need a skip since it doesn't interact with the server.